jujugui/static/gui/src/app/user/user.js

```diff
@@ -251,3 +251,25 @@ const User = class User {
 };
 
 this.jujugui.User = User;
+
+/*
```

Comment:
This is leftover from the spike, so we don't want this; we already have it from utils/github-ssh-keys.js